flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js

```diff
@@ -19,10 +19,15 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
         this.sentMessages = [];
         this.activeSystem = 'FMGC';
         this.messageQueue = [];
+        this.inFocus = false;
+        this.lastInput = 0;
     }
     get templateID() {
         return "A320_Neo_CDU";
     }
+    get isInteractive() {
+        return NXDataStore.get("MCDU_KB_INPUT", "DISABLED") === "ENABLED";
```

Comment:
What I originally meant was something like this when I talked about Boolean.
Actually have a number 0 or 1 (Boolean) stored as string ofc and when reading converting it back to a Boolean.

```js
Boolean(Number("0")));
Boolean(Number("1")));
```

Also why is it that I don't see any usage of this method??? This method should be used instead or scrapped.

Reply:
get isInteractive() is overriding an inherited function from A32NX_BaseAirliners (Line 381).

Long story short this function is called when the MCDU is first initialised (even before main loop is created) to determine if the screen should capture touch input or not.

Reply:
get isInteractive() is overriding an inherited function from A32NX_BaseAirliners (Line 381).

Long story short this function is called when the MCDU is first initialised (even before main loop is created) to determine if the screen should capture touch input or not.

Reply:
Also as for the syntax around boolean NXDataStorage items, the existing convention seems to be to use "ENABLED"/"DISABLED" as flags, also I think this is a lot more readable than trying to force boolean types, even if this would save on some string processing I don't think it's worth the loss of readability.